### ISSUE TITLE
500 General note

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#13]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/5XX.csv

**Comment by laura-ake** [comment]
@CECSpecialistI I am trying to assign this to me but I don't seem to have the power to do so.

**Comment by lake44me** [comment]
I've left the subfields 3,5,6,8, plus l, x, z (obsolete) in even though I know we have a separate issue to address them as a group.  I left a note about $3:  As with other unstructured text string notes, the materials specified could be appended to the end of the note text in parentheses with the prefix Applies to, e.g. (Applies to:  booklet)

**Comment by lake44me** [comment]
I think we talked about having a link to the Github issue in the spreadsheet but wasn't sure where it would go.  I put it in the first line Notes (Uncategorized).

**Comment by CECSpecialistI** [comment]
waiting on $5 decision

**Comment by cspayne** [comment]
checked
